apps/server/src/shared/repo/externaltool/external-tool.repo.integration.spec.ts

```diff
@@ -198,4 +219,99 @@ describe('ExternalToolRepo', () => {
 			expect(result).toMatchObject(expected);
 		});
 	});
+
+	describe('find', () => {
+		async function setupFind() {
+			const { queryExternalToolDO } = await setup();
+			queryExternalToolDO.name = '.';
+
+			const options: IFindOptions<ExternalToolDO> = {};
+
+			await em.nativeDelete(ExternalTool, {});
+			const ltiToolA: ExternalTool = externalToolFactory.withName('A').buildWithId();
```

Comment:
you can also use .buildWithId({ name: 'A' }) instead of introducing a new method, but idc